components/gripper/client_test.go

```diff
@@ -132,14 +131,6 @@ func TestClient(t *testing.T) {
 			test.That(t, spatialmath.GeometriesAlmostEqual(expectedGeometries[i], geometry), test.ShouldBeTrue)
 		}
 
-		m := gripper1Client.ModelFrame()
```

Comment:
Unless theres a reason to remove this lets keep it.  We can swap the call to ModelFrame to a call to Kinematics